src/test/java/org/springframework/samples/petclinic/integration/Browser.java

```diff
@@ -14,6 +14,10 @@ public class Browser {
 
 	public final static WebDriver webDriver = createDriver();
 
+	public static void browserClose() {
+		webDriver.close();
```

Comment:
The browser is closing after the build tool runs all the tests. However, when I run an individual test locally, the browser stays on.